ansible_runner/runner.py

```diff
@@ -286,6 +286,20 @@ def _delete(retries=15):
                         raise
                 return True
             _delete()
+
+        # Record installed Collections + Collections version info
+        cmd = 'ansible-galaxy collection list -o json'
+        proc = Popen(cmd, stdout=PIPE, stderr=PIPE, shell=True)
```

Comment:
Unless we _need it_ (and I suspect we don't), we should consider *not* doing `shell=True` here (and below):

https://docs.python.org/3/library/subprocess.html#security-considerations